go/runtime/registry/host_rofl.go

```diff
@@ -181,7 +184,10 @@ func (rh *roflHostHandler) handleHostSubmitTx(
 	switch rq.Wait {
 	case true:
 		// We need to wait for transaction inclusion.
-		rsp, err := rh.client.SubmitTxMeta(ctx, submitRq)
+		submitTxCtx, cancel := context.WithTimeout(ctx, roflSubmitTxTimeout)
```

Comment:
The caller (e.g. ROFL app itself) should do its own timeout (and it does now, I'll post a PR soon after confirming issues are fixed). This timeout is just so that in case the app doesn't, it still times out.